pyodata/v2/service.py

```diff
@@ -1337,7 +1350,12 @@ def get_entity_handler(response):
 
             entity = response.json()['d']
 
-            return EntityProxy(self._service, self._entity_set, self._entity_set.entity_type, entity)
+            try:
+                etag = response.headers.get('ETag', None)
+            except AttributeError:
```

Comment:
For the record, please explain this AttributeError catch statement or remove it.

Reply:
@bartonip This is a new blocker. @mamiksik added the except AttributeError statement but has no reliable explanation why it's needed and it makes no sense to me.